css/style.css

```diff
@@ -0,0 +1,219 @@
+
+body {
+    margin: 10px;
+    padding: 0;
+    background-color: whitesmoke;
+    font-family: Roboto, Arial, sans-serif;
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: white;
+}
+.butn {
+    height:34px;
+}
+.homeBar {
+    width: 100%;
+    overflow: auto;
+
+}
+
+.logo {
+    display: flex;
```

Comment:
- [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/))